nwb_conversion_tools/gui/nwb_conversion_gui.py

```diff
@@ -681,9 +692,13 @@ def wheelEvent(self, event):
 # If it is imported as a module
 def nwb_conversion_gui(metafile=None, conversion_module='', source_paths={},
                        kwargs_fields={}, extension_modules={}, extension_forms={},
-                       show_add_del=False):
+                       show_add_del=False, nwbfile_loc=None, conversion_class=None):
     """Sets up QT application."""
     app = QtCore.QCoreApplication.instance()
+    if conversion_class is None and conversion_module=='':
+        raise Exception('provide one of conversion_module:str or conversion_class:class')
+    if nwbfile_loc is None:
+        nwbfile_loc = os.path.join(os.getcwd(),conversion_class.__name__ + '_nwbfile_gui.nwb')
```

Comment:
What is this? A default name for the NWB file? Can you make this blank? I want this to be filled in by the user